db/schema.rb

```diff
@@ -12,7 +12,7 @@
 
 ActiveRecord::Schema[7.1].define(version: 2024_10_07_124333) do
   # These are extensions that must be enabled in order to support this database
-  enable_extension "plpgsql"
+  enable_extension "plpgsql" if Rails.env.production?
```

Comment:
I'm not sure what to do with this line
- It would get cleaned after the first next migration
- schema.rb in general is not needed in production, as it's governed by the flow of migrations
- `enable_extension "plpgsql"` is not needed in most modern systems either way (as far as I know)

I think the line should be just cleared with running `rails db:migrate` and commiting the output schema. 

What do you think @FabienChaynes ?

Reply:
I'm definitely not super knowledgeable on this topic.

What you explained made sense though, so ec1dc7dad9508b0bde5731a912a91643f6e2f13c is the result of a `rails db:migrate`.

Reply:
sure thing. I usually use either PG or SQL all the way so it's a new subject for me as well :)